src/components/common/atoms/Logo/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { LogoImg } from './styled';
+
+type Props = {
+  handleClick: () => void;
+  pic: string;
+};
+
+const Logo = ({ handleClick, pic }: Props) => {
+  return <LogoImg src={pic} onClick={handleClick} />;
```

Comment:
Once you change the props to src and onClick, you may wanna change the markup to this:
`<LogoImg {...{ src, onClick }} />`